website/docs/cloud-docs/api-docs/notification-configurations.mdx

```diff
@@ -61,11 +61,11 @@ Notifications are sent as the run progresses, and can be triggered on one or mor
 
 ## Notification Payload
 
--> **Note:** Drift detection events do not support webhooks notifications.
+The notification is an HTTP POST request with a detailed payload whose content depends on the type of notification configured.
 
-The notification is an HTTP POST request with a detailed payload describing the run. The payload depends on the type of notification configured.
+For Slack and Microsoft Teams notifications, the payload conforms to the respective webhook API, and results in a notification message with informational attachments. Refer to [Slack Notification Payloads](/cloud-docs/workspaces/settings/notifications#slack) and [Microsoft Teams Notification Payloads](/cloud-docs/workspaces/settings/notifications#microsoft-teams)  for examples. For generic notifications, the payload varies based on the type of notification:
 
-For Slack and Microsoft Teams notifications, the payload conforms to the respective webhook API, and results in a notification message with informational attachments. Refer to [Slack Notification Payloads](/cloud-docs/workspaces/settings/notifications#slack) and [Microsoft Teams Notification Payloads](/cloud-docs/workspaces/settings/notifications#microsoft-teams)  for examples. For generic notifications, the payload has several fields:
+### Run Notification Payload
```

Comment:
I'm just thinking it might be nice to have some preamble here to help reinforce the idea that there are two types of events and that this payload stuff is ONLY for generic notifications... idk thoughts?

```suggestion
### Run Notification Payload

Run events include detailed information about a specific run, including the time it began and the associated workspace and organization. Generic notifications for run events contain the following information:
```

Reply:
Yeah that makes sense to me.